NEWS.md

```diff
@@ -5,6 +5,10 @@
 
 #### NEW FEATURES
 
+#### NEW FEATURES
+
+1. `fread` no longer errors on empty files, and instead returns `NULL`, [#2898](https://github.com/Rdatatable/data.table/issues/2898). This improves compatibility with, e.g., using `fread` + `rbindlist` on a directory in which some files may be empty, since `rbindlist` already skips `NULL` elements.
```

Comment:
why not `data.table(NULL)`? consistency in returned data type is quite important. This is the reason why `drop=TRUE` arg in `[` is so bad.